examples/fungible/src/contract.rs

```diff
@@ -6,6 +6,7 @@
 mod state;
 
 use self::state::FungibleToken;
+use async_graphql::ComplexObject;
```

Comment:
Nit (mostly because @ma2bd called me out on being inconsistent on this before :), and there's an `async_graphql::SimpleObject` elsewhere in the file): I prefer to use `async_graphql::ComplexObject` (as with `Object` and `SimpleObject`) qualified, because I feel like `ComplexObject` (resp. …) isn't obviously a GraphQL thing, so it makes it a little more confusing when reading the code if you're not familiar with `async_graphql` (or just haven't used it for a while/aren't expecting to see it here).